script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -1,31 +1,31 @@
 <Page1>
-It was just like a human missile.
+She was just like a human missile.
 Arcueid descended from a height of 80 meters above Vlov.
 <Page2>
 No, it was more like she flew.
 While in the air, she took aim and angled herself before accelerating like a missile.
 <Page3>
-Sand and debris billow up into the air.
-The wind whipped up a cloud of dust that sweeps the area.
-The river of flame that had swept across the downtown area was eradicated by that single blow.
+Earth and debris billow up into the air.
+The wind whipped up a cloud of dust that scoured the area.
+The river of flame that had been flowing over the downtown area was halted by that single blow.
 <Page4>
-The policemen that surround Vlov in the distance are blown away, incapacitating most of them. They are divided into two groups: those who became unconscious after suffering a head injury and those who lose their will to fight.
+The policemen that had been encircling Vlov are blown away, and most of them are incapacitated in the process. Half of them have been knocked unconscious by blows to the head, and the rest have simply lost their will to fight.
 It's not surprising.
 <Page5>
 "In-Incre..."
-After the dust had settled, the result was in plain view.//to be rechecked "this is the result" sounds bad apparently, なにしろ
-Arcueid's rapid descent not only shattered the area around Vlov, but also a large portion of the downtown area at the north gate.
+Once the dust has settled, the extent of the destruction is plain to see. // これである reified for flow
+Arcueid's rapid descent not only shattered the area around Vlov, but also collapsed a significant portion of the business district around the north gate.
 <Page6>
 So that's what she meant by 'dropping them underground'.
-The land had sunk over 40 meters.
+The land has sunk over 40 meters.
```

Comment:
had
(again, the land had already sunk. he's viewing the aftermath now)

Reply:
i say drop the word entirely.